Code/GraphMol/AddHs.cpp

```diff
@@ -319,7 +319,8 @@ void setHydrogenCoords(ROMol *mol, unsigned int hydIdx, unsigned int heavyIdx) {
 }  // end of unnamed namespace
 
 namespace MolOps {
-void addHs(RWMol &mol, bool explicitOnly, bool addCoords) {
+void addHs(RWMol &mol, bool explicitOnly, bool addCoords,
+           const UINT_VECT *onlyOnAtoms) {
```

Comment:
Rather than starting down this road here with a single function (actually a couple of functions), I'd rather have a longer design discussion about better ways to deal with functions/methods that take a lot of optional arguments.

But this functionality doesn't need to be gated on that discussion taking place.